src/app/pages/collections/api-documentation/api-documentation-view.jsx

```diff
@@ -67,10 +67,13 @@ const useStyles = makeStyles({
   },
   pageBody: {
     display: 'grid',
-    gridTemplateColumns: '2fr 1fr',
```

Comment:
If you're changing this page, you should probably change the real-world-testing page as well, otherwise the formatting will be different. This construction is also used in the change-requests stuff, and that probably will need to change as well, unless these columns are specific to just the API Page